builtin/providers/rundeck/resource_job.go

```diff
@@ -347,22 +415,80 @@ func jobFromResourceData(d *schema.ResourceData) (*rundeck.JobDetail, error) {
 		},
 	}
 
+	// Element: Job>Dispatch
+	dispatchConfigI := d.Get("dispatch").([]interface{})
+	if len(dispatchConfigI) > 1 {
+		return nil, fmt.Errorf("rundeck command may have no more than one dispatch")
+	}
+	if len(dispatchConfigI) > 0 {
+		dispatchMap := dispatchConfigI[0].(map[string]interface{})
+		job.Dispatch.ContinueOnError = dispatchMap["continue_on_error"].(bool)
+	}
+
+	// Element: Job>Schedule
+	scheduleCron := d.Get("schedule_cron").(string)
+	if len(scheduleCron) > 0 {
+		scheduleCronArray := strings.Split(scheduleCron, " ")
+		if len(scheduleCronArray) != 7 {
+			return nil, fmt.Errorf("rundeck schedule_cron format is incorrect")
```

Comment:
Could we use a `ValidateFunc` on the `schedule_cron` attribute for this? If we do the validation there then Terraform can detect it during the "plan" step in some cases (as long as it doesn't contain any interpolation expressions).